services/blockchain-indexer/shared/database/schema/nft.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	tableName: 'nft',
+	primaryKey: 'nftID',
+	schema: {
+		nftID: { type: 'string' },
+		owner: { type: 'string' },
```

Comment:
Should we add strict not null check for safety?
```suggestion
		nftID: { type: 'string', null: false },
		owner: { type: 'string', null: false },
```